packages/manager/modules/octavia-load-balancer/src/create/controller.js

```diff
@@ -0,0 +1,15 @@
+import { PRODUCT_LINK } from './constants';
+
+export default class OctaviaLoadBalancerCreateCtrl {
+  /* @ngInject */
+  constructor(coreConfig) {
+    this.user = coreConfig.getUser();
+  }
+
+  $onInit() {
+    this.productPageLink =
+      PRODUCT_LINK[this.user.ovhSubsidiary] || PRODUCT_LINK.DEFAULT;
+
+    this.model = {};
+  }
```

Comment:
```suggestion
  constructor(coreConfig) {
    const user = coreConfig.getUser();
    this.productPageLink =
      PRODUCT_LINK[user.ovhSubsidiary] || PRODUCT_LINK.DEFAULT;
  }

  $onInit() {
    this.model = {};
  }
```
If you only use `coreConfig.getUser()` to set the page link there is no need to bind `user` to the controller.
You can also move initialization of `this.model` in the constructor (it's also ok to keep it in `$onInit`)